_posts/courses/earth-analytics-bootcamp/02-python-variables-lists/exercises/2018-07-23-python-variables-lists-01-variables.md

```diff
@@ -0,0 +1,292 @@
+---
+layout: single
+title: 'Variables in Python'
+excerpt: "Variables store data (i.e. information) that you want to re-use in your code (e.g. a single value, list of values, path to a directory, filename). Learn how to write Python Code to work with variables."
+authors: ['Jenny Palomino', 'Software Carpentry']
+category: [courses]
+class-lesson: ['python-variables-lists']
+permalink: /courses/earth-analytics-bootcamp/python-variables-lists/variables/
+nav-title: "Variables in Python"
+dateCreated: 2018-06-27
+modified: 2018-07-31
+module-title: 'Python Variables and Lists'
+module-nav-title: 'Python Variables and Lists'
+module-description: 'This tutorial teaches you how to create and manipulate variables and lists in Python. You will also learn how to plot data using the matplotlib package.'
+module-type: 'class'
+course: "earth-analytics-bootcamp"
+week: 2
+sidebar:
+  nav:
+author_profile: false
+comments: true
+order: 1
+topics:
+  reproducible-science-and-programming: ['python']
+---
+{% include toc title="In This Lesson" icon="file-text" %}
+
+In this lesson, you will write `Python` code to create variables (to store data values) and to run arithmetic calculations on variables. 
+
+<div class='notice--success' markdown="1">
+
+# <i class="fa fa-graduation-cap" aria-hidden="true"></i> Learning Objectives
+
+After completing this lesson, you will be able to:
+
+* Explain how `Python` uses variables to store data
+* Write `Python` code to:
+    * create variables to store single data values (e.g. integers, decimals, text string) 
+    * run simple arithmetic calculations on these variables (e.g. with the addition operator `+`)
+   
+ 
+# <i class="fa fa-check-square-o fa-2" aria-hidden="true"></i> What You Need
+
+Be sure that you have followed the instructions on <a href="{{ site.url }}/workshops/setup-earth-analytics-python/setup-git-bash-anaconda/" target = "_blank">Setting up Git, Bash, and Anaconda on your computer</a> to install the tools for your operating system (Windows, Mac, Linux). 
+
+Be sure that you have completed the previous lesson on <a href="{{ site.url }}/courses/earth-analytics-bootcamp/get-started-with-open-science/jupyter-notebook-interface/" target = "_blank">The Jupyter Notebook Interface</a>.
```

Comment:
@jlpalomino so here are link rules for our website.

1. if the link is EXTERNAL - this means that it takes you to another website, please open the link in a NEW BROWSER tab ie use: `target = _blank`
2. if the link is INTERNAL - thus it links to earthdatascience.org - anywhere on the site, please do not use `target=_blank` - the window can open on the same page. 

So for this PR - please update all links that are internal to earthdatascience.org to open in the same window - ie please remove target=_blank